astarte-device-conf/config.toml

```diff
@@ -0,0 +1,16 @@
+[astarte]
+connection = "mqtt" # or "grpc"
+store_directory = "/tmp/stream-rust-test/store/"
+
+# MQTT connection to Astarte
+[astarte.mqtt]
+realm = "REALM_NAME_HERE"
+device_id = "DEVICE_ID_HERE"
+pairing_url = "PAIRING_URL_HERE"
+credentials_secret = "CREDENTIALS_SECRET_HERE"
+#pairing_token = "PAIRING_TOKEN_HERE"
+astarte_ignore_ssl = false
+
+# gRPC connection to the Astarte Message Hub
+[astarte.grpc]
+endpoint = "http://[::1]:50051"
```

Comment:
This is an optional parameter, It should only be set even if the connection is `grpc`.